airbyte-config/config-models/src/main/resources/types/StandardSync.yaml

```diff
@@ -46,16 +46,38 @@ properties:
       - active
       - inactive
       - deprecated
-  # Ideally schedule and manual should be a union, but java
-  # codegen does not handle the union type properly.
-  # When schedule is defined, manual should be false.
+  # This should be populated when scheduleType is BasicSchedule.
   schedule:
     type: object
-    required:
-      - timeUnit
-      - units
     additionalProperties: false
     properties:
+      basicSchedule:
```

Comment:
based on the comments at https://github.com/airbytehq/airbyte/pull/14873/files#diff-8462fea32c1bc278ea7246bc954a7a57fbe51a81dc2b80b87b90d2c5919084b7L49-L51, can we double check the generated code is correct

Reply:
Actually removed this comment by accident; now it applies to the `basicSchedule` and `cron` properties, which I'll update.

I did check the generated code, both the basicSchedule and cron properties are fillable, which is unfortunate but aligns with the comment.